docs/amq-streams.asciidoc

```diff
@@ -0,0 +1,181 @@
+= Deploying Debezium Developer Preview with AMQ Streams
+:awestruct-layout: doc
+:linkattrs:
+:icons: font
+
+The following describes how to set up the https://developers.redhat.com/products/amq/download/[Debezium 1.0.0 Developer Preview connectors] for change data capture on Red Hat's https://www.openshift.com/[OpenShift] container platform.
+
+[NOTE]
+====
+This and other documentation pages mention ```Debezium 1.0.0 Developer Preview```, please note that this version corresponds to version ```0.10.0.Beta2``` of upstream https://github.com/debezium/debezium/tree/v0.10.0.Beta2[Debezium project].
+====
+
+== Apache Kafka Deployment
+
+For setting up Apache Kafka and Apache Kafka Connect on OpenShift, https://access.redhat.com/products/red-hat-amq#streams[Red Hat AMQ Streams] can be used, which offers "Kafka as a Service".
+It consists of an enterprise grade operator and images that bring Kafka to OpenShift.
+
+Follow these steps to prepare your Kafka cluster:
+
+* Install the AMQ Streams operator by following steps in https://access.redhat.com/documentation/en-us/red_hat_amq/7.3/html/using_amq_streams_on_openshift_container_platform/getting-started-str#downloads-str[AMQ docs].
+* Select the desired configuration and https://access.redhat.com/documentation/en-us/red_hat_amq/7.3/html/using_amq_streams_on_openshift_container_platform/getting-started-str#kafka-cluster-str[deploy your Kafka Cluster].
+* Deploy https://access.redhat.com/documentation/en-us/red_hat_amq/7.3/html/using_amq_streams_on_openshift_container_platform/getting-started-str#using-kafka-connect-with-plug-ins-str[Kafka Connect s2i].
+
+We now have a working ```Kafka cluster``` running within ```OpenShift``` with ```Kafka Connect s2i```.
+
+After a while and depending on your Kafka configuration, these pods should be running.
+
+[source%nowrap,bash]
+----
+$ oc get pods
+
+NAME                                                      READY     STATUS      RESTARTS   AGE
+debezium-kafka-cluster-entity-operator-7b6b9d4c5f-k7b92   3/3       Running     0          5m
```

Comment:
Where's the "debezium" name coming from? I think it'd help to add something like that to the sentence before:

> After a while and depending on your Kafka configuration, these pods should be running (assuming you used "debezium" as the AMQ Streams Kafka cluster name):

The contents in (...) is what I'm after, feel free to reword as you see fit.

Reply:
Done.